index.bs

```diff
@@ -10147,6 +10232,7 @@ interface AudioWorkletGlobalScope : WorkletGlobalScope {
 	readonly attribute unsigned long long currentFrame;
 	readonly attribute double currentTime;
 	readonly attribute float sampleRate;
+	readonly attribute unsigned long renderQuantumSize;
```

Comment:
This seems to affect the AudioWorklet callback quantum size?

In the spec there is the BitCrusher example, which has a comment

```
// The bitDepth parameter array has 128 sample values.
```
that probably could also updated in this PR, if that will change?

Reply:
It will indeed change the `AudioWorkletProcessor` callback buffer sizes. This allows having the best performance and lowest audio latency.

But by default, an `AudioContext` will always have 128-frames render quantum size, because we've found so much code hard-coding 128. If the `AudioContext` is however constructed with the right attributes, then the buffers might be different in size. We're simply doing `new AudioContext()` in the example, so we're guaranteed to have 128 frames here. But we can certainly rework this to not say `128 sample values`, that's probably better.

Also, in the same vein, we could add a non-normative note saying that hard-coding values when you have `buffer.length` is bad practice.

Reply:
@juj Does the response above answer your question?

Reply:
Yes it did, thanks.